src/utils.js

```diff
@@ -0,0 +1,10 @@
+exports.findHighest = (...numbs) => {
+    let highest = -Infinity;
+    for(let i = 0; i < numbs.length; i++) {
+      if(numbs[i] > highest) {
+        highest = numbs[i];
+      };
+    };
```

Comment:
Lets make of the forEach method for arrays here, so it will then look like this

```
numbers.forEach(number => {
  if (number > highest) {
    highest = number;
  }
})
```

In case you haven't use the forEach before, you can get more details [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)

Reply:
ok